presto-product-tests/README.md

```diff
@@ -407,3 +407,14 @@ been downloaded:
     # downloaded.
     presto-product-tests/conf/docker/<profile>/compose.sh pull
     ```
+
+## Possible issues
```

Comment:
Make the heading "Debugging issue" unless there is already a debugging section, if yes then add to that.